From 27662e929fb3c0faf4f37615ae677c0e400f29bf Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 22:43:36 -0400
Subject: [ethernet] cxgb4: Adds device ID for few more Chelsio T4 Adapters

Message-id: <1410475447-94392-78-git-send-email-svemuri@redhat.com>
Patchwork-id: 93890
O-Subject: [RHEL7.1 PATCH BZ 1124947 077/108] cxgb4: Adds device ID for few more Chelsio T4 Adapters
Bugzilla: 1124947
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit fb1e933d3c1a7289dc3c3456ff9b97f53ed5f1d9)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
index 9a4c152..7eadcbc 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -223,6 +223,17 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x4008, -1),
  CH_DEVICE(0x4009, -1),
  CH_DEVICE(0x400a, -1),
+ CH_DEVICE(0x400d, -1),
+ CH_DEVICE(0x400e, -1),
+ CH_DEVICE(0x4080, -1),
+ CH_DEVICE(0x4081, -1),
+ CH_DEVICE(0x4082, -1),
+ CH_DEVICE(0x4083, -1),
+ CH_DEVICE(0x4084, -1),
+ CH_DEVICE(0x4085, -1),
+ CH_DEVICE(0x4086, -1),
+ CH_DEVICE(0x4087, -1),
+ CH_DEVICE(0x4088, -1),
  CH_DEVICE(0x4401, 4),
  CH_DEVICE(0x4402, 4),
  CH_DEVICE(0x4403, 4),
@@ -235,6 +246,15 @@ static DEFINE_PCI_DEVICE_TABLE(cxgb4_pci_tbl) = {
  CH_DEVICE(0x440a, 4),
  CH_DEVICE(0x440d, 4),
  CH_DEVICE(0x440e, 4),
+ CH_DEVICE(0x4480, 4),
+ CH_DEVICE(0x4481, 4),
+ CH_DEVICE(0x4482, 4),
+ CH_DEVICE(0x4483, 4),
+ CH_DEVICE(0x4484, 4),
+ CH_DEVICE(0x4485, 4),
+ CH_DEVICE(0x4486, 4),
+ CH_DEVICE(0x4487, 4),
+ CH_DEVICE(0x4488, 4),
  CH_DEVICE(0x5001, 4),
  CH_DEVICE(0x5002, 4),
  CH_DEVICE(0x5003, 4),
-- 
1.7.1